### PR TITLE
Remove system mail option

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-users.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Feb  4 10:05:57 UTC 2016 - jreidinger@suse.com
+
+- drop receive system mail in installation (FATE#320448)
```
Comment (mvidner) [inline package/yast2-users.changes]: > Drop the Receive System Mail checkbox from the installation workflow (FATE#320448)

Comment (jreidinger) [inline package/yast2-users.changes]: thanks, will update it

**Comment by mvidner** [comment]
Just enhance the changelog note, otherwise LGTM.

**Comment by ancorgs** [comment]
If the fate is only about the dialog. Then it's fine. But have you checked if `GetRootAlias` and `SetRootAlias` are also used in autoyast or have a default value set by control.xml or something like that?  I think the plan was to also remove all the dead code about that feature.

**Comment by jreidinger** [comment]
related pr  https://github.com/yast/yast-storage/pull/187 which removes one of removed method from other module where it doesn't make sense.

**Comment by ancorgs** [comment]
Shouldn't we do something to ensure that this not get installed alongside an old version of yast2-storage (one requiring `UsersSimple.AfterAuth`to be there)?

**Comment by ancorgs** [comment]
Nice to see so much code from `UsersSimple` being dropped. Do we have some mechanism to announce these API changes? I wouldn't be unlikely to have a third party module (let's say SAP stuff) using it. We should warn them somehow.

**Comment by jreidinger** [comment]
@ancorgs We can add conflict. so I do it.

regarding API changes we do not have such mechanism but as UsersSimple is installation only ( for runtime usage there is Users module ), I do not expect much problems.

**Comment by ancorgs** [comment]
LGTM

About the deprecation mechanism (well, "dropping" is a better word than "deprecation"), I will start a discussion in yast-devel

**Comment by kobliha** [comment]
:clap:

**Pull request merged**